### PR TITLE
fastscan improvements

### PR DESCRIPTION
**Pull request by Glenn-1990**

### CODE REVIEW

**Comment by perexg** [comment]
The code looks good.. Thanks.

Could you replace "S2" with  "delsys" (delivery system) and use dvb_str2delsys() / dvb_delsys2str() ?

Note that dvb_str2delsys() accepts also lowercase strings (dvbs,dvbs2)..

**Comment by Glenn-1990** [comment]
updated ;-)

File: data/conf/fastscan [hunk 1]
```diff
@@ -1,86 +1,128 @@
...
   },
   {
-    "name": "Canal Digitaal Astra3",
+    "name": "Canal Digitaal HD",
     "position": 235,
     "frequency": 12187000,
-    "pid": 900
+    "symbolrate" : 27500000,
+    "polarisation" : "H",
+    "S2" : 1,
```
Comment (perexg) [inline data/conf/fastscan]: Ommited S2 / delsys change ?

Comment (Glenn-1990) [inline data/conf/fastscan]: fixed ;-)

**Comment by perexg** [comment]
Merged. Thanks.

**Pull request closed without merging**